RENOVATE.md

```diff
@@ -141,6 +141,7 @@ alerts, enabled by a previous preset, will use a `rangeStrategy` of `update-lock
 ```json
 "packageRules": [
     {"matchDepTypes": ["require"], "rangeStrategy": "widen"},
```

Comment:
We may also want to add "dependencies" to the "matchDepTypes" array here but I can do that in another PR.